service/src/main/kotlin/fi/espoo/evaka/incomestatement/IncomeStatementController.kt

```diff
@@ -96,6 +97,6 @@ class IncomeStatementController(
         Audit.IncomeStatementsAwaitingHandler.log()
         accessControl.requirePermissionFor(user, Action.Global.FETCH_INCOME_STATEMENTS_AWAITING_HANDLER)
         val areasList = areas.split(",").filter { it.isNotEmpty() }
-        return db.read { it.fetchIncomeStatementsAwaitingHandler(areasList, page, pageSize) }
+        return db.read { it.fetchIncomeStatementsAwaitingHandler(LocalDate.now(), areasList, page, pageSize) }
```

Comment:
You should use `HelsinkiDateTime.now().toLocalDate()` so that server timezone does not matter.

Reply:
Fixed!